### PR TITLE
docs: readability improvements r2

### PR DESCRIPTION
**Pull request by ryanrussell**
Signed-off-by: Ryan Russell <git@ryanrussell.org>

Round 2 on docs readability fixes. Completes `.md` things I noticed.

One thing not included in this PR -- I'm not sure what you are going for on Line 77 of `forecasting_overview.md` with `especially for more expre based models` ... maybe someone internal could touch up that sentence?

Best,
Ryan

### CODE REVIEW

**Review by gdevos010 — APPROVED** [review]

File: docs/userguide/forecasting_overview.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ These models are shown with a "✅" under the `Multivariate` column on the [mode
 
 Some models support being fit on multiple time series. To do this, it is enough to simply provide a Python `Sequence` of `TimeSeries` (for instance a list of `TimeSeries`) to `fit()`. When a model is fit this way, the `predict()` function will expect the argument `series` to be set, containing
 one or several `TimeSeries` (i.e., a single or a `Sequence` of `TimeSeries`) that need to be forecasted. 
-The advantage of training on multiple series is that a single model can be exposed to more patterns occuring across all series in the training dataset. That can often be beneficial, especially for more expre based models.
+The advantage of training on multiple series is that a single model can be exposed to more patterns occurring across all series in the training dataset. That can often be beneficial, especially for more expre based models.
```
Comment (hrzn) [inline docs/userguide/forecasting_overview.md:77]:
```suggestion
The advantage of training on multiple series is that a single model can be exposed to more patterns occurring across all series in the training dataset. That can often be beneficial, especially for more bigger models with more capacity.
```

**Review by hrzn — APPROVED** [review]
Thanks, just added a small suggestion in one of the files.

**Comment by ryanrussell** [comment]
> Thanks, just added a small suggestion in one of the files.

Did one more force push on this to make `more bigger` -> `larger` :)

**Comment by hrzn** [comment]
> > Thanks, just added a small suggestion in one of the files.
> 
> Did one more force push on this to make `more bigger` -> `larger` :)

Oh my 🤦 thanks!

**Pull request merged**